scripts/buildTypes.mjs

```diff
@@ -111,7 +110,7 @@ async function main() {
     );
   }
 
-  await exec(['pnpm', 'tsc', '-b', tsconfigPath].join(' '));
```

Comment:
@michaldudak This is hard to debug, I'm replacing with `execa` and having it dump its output on the console. Are you ok with introducing this dependency?

Reply:
Sure, I don't mind, especially if it's been used in other repos already.